app/javascript/packages/form-steps/form-error.ts

```diff
@@ -16,11 +18,18 @@ class FormError extends Error {
 
   isDetail: boolean;
 
-  constructor(message?: string, options?: FormErrorOptions) {
+  messageProcessor?: (message: string) => string | ReactNode;
+
+  constructor(
+    message?: string,
+    options?: FormErrorOptions,
+    messageProcessor?: (message: string) => string | ReactNode,
```

Comment:
Could this be added in the existing `options` object rather than adding a third parameter?

Reply:
@aduth , moved to options and it seems working. Thanks!